helm/hpcc/templates/_helpers.tpl

```diff
@@ -825,6 +825,24 @@ A kludge to ensure until the mount of a PVC appears (this can happen with some t
       mountPath: {{ .volumePath | quote }}
 {{- end }}
 
+{{/*
+Inject container to perform any post plane initialization validation
+Pass in dict with volumeName, volumePath and cmds
+*/}}
+{{- define "hpcc.validatePlaneScript" -}}
+- name: {{ printf "validate-plane-script-container-%s" .volumeName }}
+  {{- include "hpcc.addImageAttrs" . | nindent 2 }}
+  command: ["/bin/bash", "-c"]
+  args:
+  - |
+{{- range $cmd := .cmds }}
+    {{ $cmd }}
```

Comment:
Should there be a ; after each command?

Reply:
no, it's because the block scalar passes it as a script (complete with line breaks) which bash interprets as individual commands (as it would in a script.
e.g. it's the equivalent of:
```
/bin/bash -c "
echo 'command 1'
echo 'command 2'
echo 'command 3'
"
```

The example in the JIRA is generated as:
```
          initContainers:
          - args:
            - |
              echo "Hello, World!"
              ls -l
              df -h
              end_time=$((SECONDS + 1200)); while [ $SECONDS -lt $end_time ]; do if mountpoint -q '/var/lib/HPCCSystems/hpcc-data'; then device=$(df /var/lib/HPCCSystems/hpcc-data | awk 'NR==2 {print $1}'); if [[ "$device" == "/dev/md0" ]]; then echo "correct mount device"; exit 0; else echo "wrong mount device \"$device\", will look again!"; fi; fi; echo waiting for mount point; sleep 5; done; echo failed; exit 1
            command:
            - /bin/bash
            - -c
````


.. and executes as expected.